src/assets/img/icons/mediatype/account.ts

```diff
@@ -0,0 +1,14 @@
+import { svg } from 'lit';
```

Comment:
I think we have these icons in iaux-icon repo. can we use those?

Reply:
The media-type icon has a few customizations to it that the iaux-icons don't handle and would've be a lot of work to update so going to use these for now. It would be nice to pull out the media-type icon to iaux-icons (or somewhere) since we have a few spots on the site where we use these.